S1_NRB/nrb.py

```diff
@@ -83,6 +83,7 @@ def format(config, product_type, scenes, datadir, outdir, tile, extent, epsg, wb
         - np: noise power (NESZ, per polarization)
         - gs: gamma-sigma ratio: sigma0 RTC / gamma0 RTC
         - sg: sigma-gamma ratio: gamma0 RTC / sigma0 ellipsoidal
+        - wm: OCN product wind model; requires OCN scenes via argument `scenes_ocn`
```

Comment:
Just saw that the recently introduced `ld`-layer is missing here. Might as well add it with this PR?

Reply:
done: [a50f724](https://github.com/SAR-ARD/S1_NRB/pull/138/commits/a50f724363d721ba951909ff2c2c7fa819f91b99)